client/state/action-types.js

```diff
@@ -935,6 +935,10 @@ export const REWIND_BACKUP_PROGRESS_REQUEST = 'REWIND_BACKUP_PROGRESS_REQUEST';
 export const REWIND_BACKUP_REQUEST = 'REWIND_BACKUP_REQUEST';
 export const REWIND_BACKUP_UPDATE_ERROR = 'REWIND_BACKUP_UPDATE_ERROR';
 export const REWIND_BACKUP_UPDATE_PROGRESS = 'REWIND_BACKUP_UPDATE_PROGRESS';
+export const REWIND_BACKUPS_RECEIVE = 'REWIND_BACKUPS_RECEIVE';
```

Comment:
Let's avoid the `RECEIVE` naming pattern, it's really not meaningful. Are we adding rewind records here for example?

Reply:
Renamed to `SET` in 2094e19.